rest-service/manager_rest/deployment_update/handlers.py

```diff
@@ -184,6 +187,51 @@ def modify(self, ctx, current_entities):
                 self._modify_relationship_operation,
                 self._modify_node_operation)
 
+    def _update_stored_operations(self, ctx, node, new_operation):
+        """Update the operations table with the new operation inputs."""
+
+        def _filter_operation(column):
+            # path in the parameters dict that stores the node name
+            node_name_path = ('task_kwargs', 'cloudify_context', 'node_name')
+            # this will use postgres' json operators
+            return cast(column, JSON)[node_name_path].astext == node.id
+
+        if not ctx.deployment_update.update_executions:
+            return
+        executions = self.sm.list(models.Execution, filters={
+            'deployment_id': ctx.deployment_update.deployment_id,
+            'status': [
+                ExecutionState.PENDING,
+                ExecutionState.STARTED,
+                ExecutionState.CANCELLED,
+                ExecutionState.FAILED
+            ]
+        })
+        graphs = self.sm.list(models.TasksGraph, filters={
+            'execution_id': [e.id for e in executions]
+        })
```

Comment:
unfortunate that we have to do essentially joins on the python side, but that's what the storage-manager makes us do...